protobuf/models/value.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+// TODO(adrian@preemo.io, 03/08/2023): figure out a better name for this message
+message Value {
+  // If value is not set, that indicates a null value.
+  oneof value {
```

Comment:
[the struct well known type](https://github.com/protocolbuffers/protobuf/blob/a779ae609c9ca6fa37ee4acec2d48eaa9c3afb91/src/google/protobuf/struct.proto) has a similar thing going on, if that's helpful at all